subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -330,6 +330,19 @@ In order to disable auto-provisioning, you can use the `org.gradle.java.installa
 * Either start gradle using `-Porg.gradle.java.installations.auto-download=false`
 * Or put `org.gradle.java.installations.auto-download=false` into a `gradle.properties` file.
 
+[[sub:removing_auto_provisioned_toolchain]]
+=== Removing an auto-provisioned toolchain
+
+In some cases, a user might want explicit control over the toolchain configuration for their project.
```

Comment:
This line and the next one do not provide any new information.
The section is in the toolchains chapter so there is no need to re-iterate what toolchains provide.

Please remove this line and the next one.